### PR TITLE
Add product to Getting started and update Admins instructions

### PR DESCRIPTION
**Pull request by sblausten**
https://app.shortcut.com/larder/story/22408/assigning-admins-refers-to-old-way-in-marketting-site

### CODE REVIEW

File: content/docs/getting-started/assigning-admins/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Roadie comes with a predefined set of Roles to use for access control to core app functionality. Custom roles can be configured via the Role Management tab using our predefined set of permissions. 
+
+![Adding roles to a user](./add-roles.png)
+
+## Non-Admins
+
+Non-admins will not be able to access any app level configuration like editing plugin layouts, configuring new plugins or defining plugin config. 
+
+You can view the permissions for the Admin group via the Role Management tab in the Administration section.
```
Comment (punkle) [inline content/docs/getting-started/assigning-admins/index.md:32]: We could link to this page at the bottom here I think https://roadie.io/docs/details/permissions/

File: content/docs/integrations/jenkins/index.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ This page explains how to configure it in Roadie Backstage.
 
 ## Prerequisites
 
-1. You must be an admin in Roadie. By default, all users are admins. Learn how to designate certain users as admins [here](/docs/getting-started/create-admin-group/).
+1. You must be an admin in Roadie. By default, all users are admins. Learn how to designate certain users as admins [here](/docs/getting-started/assigning-admins/).
```
Comment (punkle) [inline content/docs/integrations/jenkins/index.md:31]:
```suggestion
1. You must be an admin in Roadie. By default, the first Roadie user is an admin. Learn how to designate certain users as admins [here](/docs/getting-started/assigning-admins/).
```

**Review by punkle — APPROVED** [review]

File: content/docs/getting-started/assigning-admins/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Designating admins
+publishedDate: '2023-08-31'
```
Comment (Xantier) [inline content/docs/getting-started/assigning-admins/index.md:3]:
```suggestion
publishedDate: '2024-08-19'
```

File: content/docs/getting-started/create-admin-group/index.md
Comment (Xantier) [inline content/docs/getting-started/create-admin-group/index.md:1]: Add a redirection of this deleted path on the config file in the root of the repo

**Review by Xantier — APPROVED** [review]

**Pull request merged**